site/documentation/content/api/telemetry/index.md

```diff
@@ -39,9 +39,9 @@ All other combinations are not supported by Hono and may result in the terminati
 
 As indicated above, it is up to the discretion of the protocol adapter whether it wants to use *AT LEAST ONCE* or *AT MOST ONCE* delivery semantics.
 
-Hono's HTTP adapter allows devices to indicate, which delivery semantics they want to use when uploading telemetry data.
+Hono's HTTP adapter allows devices to indicate, which delivery semantics they want to use when forwarding telemetry data.
 
-The following sequence diagram illustrates the flow of messages involved in the *HTTP Adapter* forwarding an *unsettled* telemetry data message to the downstream AMQP Messaging Network implementing *AT MOST ONCE* delivery semantics.
+The following sequence diagram illustrates the flow of messages involved in the *HTTP Adapter* forwarding an *settled* telemetry data message to the downstream AMQP Messaging Network implementing *AT MOST ONCE* delivery semantics.
```

Comment:
no, we are indeed always sending unsettled messages. The adapter simply doesn't wait for the outcome of the transfer before sending back a 202 to the device ... please revert

Reply:
I had to read the passage in the documentation several times to understand what was meant. Maybe it should be made more explicit that the HTTP adapter implements both delivery semantics with *unsettled* mode. 

We could change the sentence right before this from:

> Hono's HTTP adapter allows devices to indicate, which delivery semantics they want to use when forwarding telemetry data.

to:

> Hono's HTTP adapter allows devices to indicate, which delivery semantics they want to use when forwarding telemetry data; the message is always forwarded as *unsettled*.

WDYT?

Reply:
> Hono's HTTP adapter allows devices to indicate which delivery semantics they want to use when uploading telemetry data. The HTTP adapter always forwards messages *unsettled* and either ignores the outcome of the message transfer (AT_MOST_ONCE) or waits for the downstream peer to accept the message (AT_LEAST_ONCE)  before acknowledging the reception of the message to device.

WDYT?